src/sidebar/components/annotation-body.js

```diff
@@ -9,82 +13,78 @@ import MarkdownView from './markdown-view';
  * Display the rendered content of an annotation.
  */
 export default function AnnotationBody({
-  collapse,
+  annotation,
   isEditing,
-  isHiddenByModerator,
-  onCollapsibleChanged,
   onEditText,
-  onToggleCollapsed,
   text,
 }) {
+  // Should the text content of `Excerpt` be rendered in a collapsed state,
+  // assuming it is collapsible (exceeds allotted collapsed space)?
+  const [isCollapsed, setIsCollapsed] = useState(true);
+
+  const onChangeCollapsed = toCollapsed => {
+    setIsCollapsed(toCollapsed);
+  };
```

Comment:
I wasn't sure if it was kosher to pass around references to the state-setting functions that belong to the component—is it?

Reply:
I'm not aware of any recommendations against it. The only theoretical issue I can see is that the behavior changes if the state setter is called with a function rather than a value ([see note](https://reactjs.org/docs/hooks-reference.html#functional-updates)). I don't see that being an issue here.